src/assisted_test_infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -599,6 +600,40 @@ def set_ipxe_url(self, network_name: str, ipxe_url: str):
         self.destroy_network(network)
         self.create_network(xml.toprettyxml())
 
+    def _create_dns_host_xml(self, api_vip: str, cluster_name: str):
+        """
+        Creates an XML objet for adding a DNS host to a libvirt network
+        """
+        host_element = Element("host", ip=api_vip)
+        if cluster_name != "":
+            hostname = f"api.{cluster_name}.{self._entity_config.base_dns_domain}"
```

Comment:
I might be missing some details here, but don't we have the same configuration in the terraform template?
For example: https://github.com/openshift/assisted-test-infra/blob/73801be515378e3b60060d8a7a625a36e424ab5e/terraform_files/baremetal/main.tf#L166

Reply:
I wasn't aware that the terraform variables could be passed to this controller! Let me see if I can access it or pass it into this function.